README.md

```diff
@@ -60,31 +58,35 @@ Also see the
 [distributed_key_generation](https://github.com/poanetwork/threshold_crypto/blob/d81953b55d181311c2a4eed2b6c34059fcf3fdae/src/poly.rs#L967)
 test.
 
-## More Details
+### Environment Variables
 
-The basic usage outline is: choose a threshold value t, create a key set, then
-distribute N secret key shares among the participants and publish the public
-master key. A third party can now encrypt a message to the public master key
+[_MLOCK_SECRETS:_](https://github.com/poanetwork/threshold_crypto/blob/master/src/lib.rs#L51) Enabled by default. We recommend only disabling mlock (MLOCK_SECRETS=false) for development and testing purposes when there is no ability to increase the locked memory limit. When mlock is disabled, unmanaged copies of secret keys remain in memory.
```

Comment:
I would change this paragraph to something like:

[`MLOCK_SECRETS`](https://github.com/poanetwork/threshold_crypto/blob/master/src/lib.rs#L51): Set's whether or not the Unix syscall `mlock` or WinAPI function `VirtualLock()` will be called on portions of memory which contain secret values. This option is enabled by default (`MLOCK_SECRETS=true`). Disabling memory locking (`MLOCK_SECRETS=false`) allows secret values to be copied to disk, where they will not be zeroed on drop and may persist on disk indefinitely. **Disabling memory locking should only be done in development and testing.** Disabling memory locking is useful because it removes the possibility of tests failing due to reaching the testing system's locked memory limit (e.g. if you are writing a crate that uses `threshold_crypto` and you write a test that maintains hundreds or thousands of secrets in memory simultaneously, you run the risk of reaching your system's allowed number of locked pages, which will cause this library to fail).